app/notifications/process_notifications.py

```diff
@@ -125,6 +125,9 @@ def persist_notification(
         if key_type != KEY_TYPE_TEST:
             if redis_store.get(redis.daily_limit_cache_key(service.id)):
                 redis_store.incr(redis.daily_limit_cache_key(service.id))
+            if notification_type == SMS_TYPE and redis_store.get(redis.sms_daily_count_cache_key(service.id)):
+                num_message_parts = create_content_for_notification(template, personalisation).fragment_count
+                redis_store.incrby(redis.sms_daily_count_cache_key(service.id), num_message_parts)
```

Comment:
Should we split this off into its own method for easier maintainability?

Reply:
you mean for`create_content_for_notification(template, personalisation).fragment_count` ? - that's a good idea, we do it three times here.